pagerduty/event_orchestration_path.go

```diff
@@ -65,6 +65,7 @@ type EventOrchestrationPathRuleActions struct {
 	EventAction                string                                             `json:"event_action"`
 	Variables                  []*EventOrchestrationPathActionVariables           `json:"variables"`
 	Extractions                []*EventOrchestrationPathActionExtractions         `json:"extractions"`
+	EscalationPolicy           string                                             `json:"escalation_policy,omitempty"`
```

Comment:
I think we need to drop `omitempty` here because `omitempty` is used to ignore a property with falsy values when serializing a struct into JSON. If we keep it here it won't allow us to delete the EP action by setting it to null.
```suggestion
	EscalationPolicy           string                                             `json:"escalation_policy"`
```

Reply:
That's a good point. I accepted your suggestion